includes/class-wcpdf-settings-debug.php

```diff
@@ -348,6 +373,66 @@ public function import_settings( $data ) {
 		}
 	}
 	
+	public function reset_settings( $data ) {
+		extract( $data );
+		
+		if ( empty( $type ) ) {
+			$message = __( 'Reset settings type is empty!', 'woocommerce-pdf-invoices-packing-slips' );
+			wcpdf_log_error( $message );
+			wp_send_json_error( compact( 'message' ) );
+		}
+		
+		$settings_option = '';
+		
+		switch ( $type ) {
+			case 'general':
+				$settings_option = 'wpo_wcpdf_settings_general';
+				break;
+			case 'debug':
+				$settings_option = 'wpo_wcpdf_settings_debug';
+				break;
+			default:
+				$settings_option = apply_filters( 'wpo_wcpdf_reset_settings_option', $settings_option, $type );
+				break;
+		}
+		
+		// maybe it's a document type settings request
+		if ( empty( $settings_option ) ) {
+			$documents = WPO_WCPDF()->documents->get_documents( 'all' );
+			foreach ( $documents as $document ) {
+				if ( $type == $document->slug ) {
+					$settings_option = "wpo_wcpdf_documents_settings_{$document->get_type()}";
+					break;
+				}
+			}
+			
+			if ( empty( $settings_option ) ) {
+				$message = __( 'Reset settings key is empty!', 'woocommerce-pdf-invoices-packing-slips' );
+				wcpdf_log_error( $message );
+				wp_send_json_error( compact( 'message' ) );
+			}
+		}
+		
+		$updated = update_option( $settings_option, [] );
+		if ( $updated ) {
+			$message = sprintf(
+				/* translators: settings type */
+				__( '%s settings reset successfully!', 'woocommerce-pdf-invoices-packing-slips' ),
+				$type
+			);
+			wcpdf_log_error( $message, 'info' );
+			wp_send_json_success( compact( 'type', 'message' ) );
+		} else {
+			$message = sprintf(
+				/* translators: settings type */
+				__( 'An error occurred when trying to reset the %s settings.', 'woocommerce-pdf-invoices-packing-slips' ),
```

Comment:
This error message is triggered when you try to delete twice a settings option. It's a similar case to which I commented when you added the export/export feature ([here](https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/pull/473#discussion_r1095123603)), but the other way around:

![image](https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/assets/38109855/aa757a94-a950-460a-9372-020d257c6fc2)

But more than an error, this should be a notice, letting know the users that the settings option was already reset.